subsys/dfu/boot/mcuboot.c

```diff
@@ -37,6 +37,12 @@
 #define BOOT_FLAG_UNSET 3
 #define BOOT_FLAG_ANY   4  /* NOTE: control only, not dependent on sector */
 
+/*
+ * The slot from which mcuboot boots from.
+ * Currently only used by imgmgr.
+ */
+int boot_current_slot;
```

Comment:
So, I think the original idea was that images could run on any slot, at least to be implemented some day; this was never done and maybe never will. This is could as well be removed and in the `mcumgr` repo references to to this variable could be changed to `0`.

Reply:
^^ If changing, better to a macro defined value. Actually I have request for any image boot in my backlog.

Reply:
`boot_current_slot` was now removed and a macro was defined in order to be used on img_mgmt.c